### PR TITLE
Update hotfixes-2-18.md

### PR DESCRIPTION
**Pull request by SherinDaher-Runai**

### CODE REVIEW

File: docs/home/changelog/hotfixes-2-18.md [hunk 1]
```diff
@@ -12,28 +12,46 @@ The following is a list of the known and fixed issues for Run:ai V2.18.
...
+| RUN-23909 | 2.18.73 | Fixed an issue where users based on group permissions cannot see dashboards. |
+| RUN-23857 | 2.18.72 | Dashboard to transition from Grafana v9 to v10. |
+| RUN-24010 | 2.18.72 | Fixed an infinite loop issue in the cluster-sync service. |
+| RUN-23040 | 2.18.72 | Fixed an edge case where the Run:ai container toolkit hangs when user is spawning hundreds of sub-processes. |
+| RUN-23802 | 2.18.70 | Fixed an issue where new scheduling rules were not applied to existing workloads, if those new rules were set on existing projects which had no scheduling rules before. |
+| RUN-23211 | 2.18.70 | Fixed an issue where workloads were stuck at "Pending" when the command-line interface flag --gpu-memory was set to zero. |
+| RUN-23778 | 2.18.68 | Fixed an issue where in single-sign-on configuration, the mapping of UID and other properties would sometimes disappear. |
+| RUN-23762 | 2.18.68 | Fixed an issue where the wrong version of a Grafana dashboard was displayed in the UI. |
+| RUN-21198| 2.18.66 | Fixed an issue where creating a training workload via yaml (kubectl apply -f) and specifying spec.namePrefix, created infinite jobs. |
```
Comment (itayvallach) [inline docs/home/changelog/hotfixes-2-18.md:31]: missing space after RUN-21198

File: docs/home/changelog/hotfixes-2-18.md [hunk 2]
```diff
@@ -12,28 +12,46 @@ The following is a list of the known and fixed issues for Run:ai V2.18.
...
+| RUN-23541 | 2.18.65 | Fixed an issue where in some cases workload authorization did not work properly due to wrong oidc configuration. |
+| RUN-23291 | 2.18.64 | CLI change text to be user friendly. |
```
Comment (itayvallach) [inline docs/home/changelog/hotfixes-2-18.md:33]: Let's remove this line altogether.

Comment (SherinDaher-Runai) [reply docs/home/changelog/hotfixes-2-18.md:33]: @itayvallach 2.18.64?

Comment (itayvallach) [reply docs/home/changelog/hotfixes-2-18.md:33]: remove line 33.

**Review by itayvallach — APPROVED** [review]

**Pull request merged**